### DISCUSSION TITLE
Linux Support?

### DISCUSSION THREAD

**Discussion by chrisspen** [arkypita/LaserGRBL#2083] [General]
Has anyone investigated Linux support in a while?

I saw the FAQ mention of PlayOnLinux and a direct guide to setting up Wine, but neither of these worked for me. The PoL GUI just gave me a bunch of useless "An error has happened. Click next." and never showed me the actual error nor ever launched a GUI.

The direct Wine instructions failed for me at `winecfg`, as every single needed dll is missing despite the packages installing just fine.

**Comment by nusse** [comment]
The problem is somehow the installer. I installed LaserGRBL using a WinXP-VM and then just copied the installed files.
